making a new top level section to work with DataFrames (#2717)
JuliaData/DataFrames.jl: docs/src/index.md

other great libraries for certain use-cases -- but it provides great data
wrangling functionality through a familiar interface.

> TO understand the toolchain in more details, have a look at the tutorials in this manual. **It is highly recommended that new users start with the [First Steps with DataFrames](@ref) tutorial.**

Reviewer: ```suggestion
To understand the toolchain in more details, have a look at the tutorials in this manual. **It is highly recommended that new users start with the [First Steps with DataFrames](@ref) tutorial.**
```

> To understand the toolchain in more details, have a look at the tutorials in this manual. **It is highly recommended that new users start with the [First Steps with DataFrames](@ref) tutorial.**

Reviewer: ```suggestion
To understand the toolchain in more details, have a look at the tutorials in this manual. New users can start with the [First Steps with DataFrames](@ref) tutorial in this manual.
```

other great libraries for certain use-cases -- but it provides great data
wrangling functionality through a familiar interface.

> To understand the toolchain in more details, have a look at the tutorials in this manual. New

Reviewer: ```suggestion
To understand the toolchain in more detail, have a look at the tutorials in this manual. New
```

> users can start with the [First Steps with DataFrames.jl](@ref) tutorial in this manual.

Reviewer: ```suggestion
users can start with the [First Steps with DataFrames.jl](@ref) section.
```

```@index
Pages = ["lib/types.md", "lib/functions.md"]
```

## Supporting and Citing

Much of the frameworks and softwares in this ecosystem was developed as part of the of
academic research. If you would like to help support it, please star the repository as
such metrics may help us secure funding in the future. If you use our framework as part
of your research, teaching, or other activities, we would be grateful if you would start

> and fork it.

Reviewer: why do you think it is needed to be added at this point?

Author: Ok, we can remove it. My purpose was to cite it
